<launch>
    <arg name="dir_map" default="/media/spcomen21/WD5T/Experiment/LiDAR-200/kitti/06/map/" />

    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find lidar_200)/rviz_cfg/relocate_200.rviz"/>

    <node name="load_map" pkg="lidar_200" type="load_map" output="screen"> 
        <param name="PARA_DIR_MAP" value="$(arg dir_map)" />
    </node>

    <node name="realtime_locate" pkg="lidar_200" type="realtime_locate" output="screen"> 
        <param name="PARA_DIR_MAP" value="$(arg dir_map)" />
        <param name="PARA_MIN_Z_LOCATE" type="double" value="-0.5" />
        <param name="PARA_MAX_Z_LOCATE" type="double" value="50.0" />
        <param name="PARA_LENGTH" type="double" value="1.0" />
    </node>
</launch>


